### ISSUE TITLE
Move FAQs to the GTN

### ISSUE THREAD

**Issue by beatrizserrano** [galaxyproject/galaxy-hub#1262]
This issue keeps track of the [FAQs in the Community Hub](https://galaxyproject.org/support/) moved to the [GTN FAQs](https://training.galaxyproject.org/training-material/faq).

### Basics
- [ ] [Getting an account at Galaxy Main (https://usegalaxy.org)](https://galaxyproject.org/support/account/)
- [ ] [Loading Data](https://galaxyproject.org/support/loading-data/)
- [ ] [Downloading Data](https://galaxyproject.org/support/download-data/)  
  - [ ] Download entire histories (@Melkeb)
  - [ ] Download datasets (@Melkeb)
  - [ ] Dataset downloading incompletely? (@Melkeb)
  - [ ] Dataset(s) not downloading at all? (@Melkeb)
- [ ] [Finding tools](https://galaxyproject.org/support/finding-tools/) (@garimavs)
- [ ] [Where is the tool help?](https://galaxyproject.org/support/tool-help/)
- [ ] [Job details: bug, info, and other useful dataset icons](https://galaxyproject.org/support/dataset-icons/) (@garimavs)

### Datasets and Histories
- [X] [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)
- [ ] [Managing Datasets](https://galaxyproject.org/learn/managing-datasets/)
- [ ] [Datasets and how jobs execute](https://galaxyproject.org/support/how-jobs-execute/) 
  - [ ] Dataset status (@garimavs)
- [ ] [More about moving Datasets and Histories](https://genomicsvirtuallab.wordpress.com/2016/07/13/moving-data-between-galaxy-instances/) (external GVL blog)  (@Melkeb)

### Data Options
- [ ] [Sharing and Publishing your work](https://galaxyproject.org/learn/share/) (@NdzeRegina)
- [ ] [Data Privacy Features](https://galaxyproject.org/learn/privacy-features/) (@Melkeb)
- [X] [Moving data between Galaxy servers (any!)](https://genomicsvirtuallab.wordpress.com/2016/07/13/moving-data-between-galaxy-instances/)

### Help Guides
- [X] [Why develop tools for Galaxy?](https://galaxyproject.github.io/why-galaxy)
- [X] [Galaxy Tricks](https://github.com/bgruening/galaxy-tricks/blob/master/README.md)
- [X] [Commercial Galaxy Support](https://galaxyproject.org/support/commercial/)

### Account
- [ ] [My history is missing! Good news, it probably isn't.](https://galaxyproject.org/support/missing-history/#my-history-is-missing-)  (@Melkeb)
- [ ] [My dataset is missing](https://galaxyproject.org/support/missing-history/#my-history-is-missing-) (@Melkeb)
- [ ] [A history was shared with me, where can I find it?](https://galaxyproject.org/support/missing-history/)
- [ ] [The account usage quota seems incorrect](https://galaxyproject.org/support/account-quotas/)
- [ ] [Checking for active vs deleted vs permanently deleted (purged) datasets and histories](https://galaxyproject.org/learn/managing-datasets/#delete-vs-delete-permanently)
- [ ] [Reset password or Change email, username, password](https://galaxyproject.org/support/account-updates/) (@Melkeb)

### Unexpected results
- [ ] [My job ended with an error. What can I do?](https://galaxyproject.org/support/tool-error/)
  - [ ] [Determining the job failure type](https://galaxyproject.org/support/tool-error/#determining-the-job-failure-type) (@garimavs)
- [ ] [Reporting Usage Issues or Software bugs](https://galaxyproject.org/issues/) (@garimavs)
- [ ] [Known Issues - Galaxy](https://github.com/galaxyproject/galaxy/issues/)
- [ ] [Known Issues - Main https://usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook/issues/)
- [ ] [Known Issues - Tools-devteam](https://github.com/galaxyproject/tools-devteam/issues/)
- [ ] [Known Issues - Tools-iuc](https://github.com/galaxyproject/tools-iuc/issues)

### Getting inputs right
- [ ] [How to format fastq data for tools that require .fastqsanger format?](https://galaxyproject.org/support/fastqsanger/)
- [ ] [Understanding compressed fastq data (fastq.gz)](https://galaxyproject.org/support/compressed-fastq/)
- [ ] [Reformatting fastq data loaded with NCBI SRA](https://galaxyproject.org/support/ncbi-sra-fastq/)
- [ ] [Format help for Tabular/BED/Interval Datasets](https://galaxyproject.org/support/tabular/)
- [ ] [Common datatypes explained](https://galaxyproject.org/learn/datatypes/)
- [ ] [The tool I'm using does not recognize any input datasets. Why?](https://galaxyproject.org/support/datatypes-and-tools/)  (@Melkeb)
- [ ] [How do I find, adjust, and/or correct metadata?](https://galaxyproject.org/support/metadata/)
- [ ] [Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/)
- [ ] [Preparing and using a Custom Reference Genome or Build](https://galaxyproject.org/learn/custom-genomes/)
- [ ] [Mismatched Chromosome identifiers (and how to avoid them)](https://galaxyproject.org/support/chrom-identifiers/)
- [ ] [Extended Help for Differential Expression Analysis Tools](https://galaxyproject.org/support/diff-expression/)

**Comment by garimavs** [comment]
Hello @beatrizserrano, I would like to work on the remaining part of [Dataset and how jobs execute](https://galaxyproject.org/support/how-jobs-execute/) and [A history was shared with me, where can I find it?](https://galaxyproject.org/support/missing-history/). Kindly add my name, if possible!


I also think that we can cover [Checking for active vs deleted vs permanently deleted (purged) datasets and histories](https://galaxyproject.org/learn/managing-datasets/#delete-vs-delete-permanently) under [Managing Datasets](https://galaxyproject.org/learn/managing-datasets/) as it is its sub-part.

**Comment by beatrizserrano** [comment]
> Hello @beatrizserrano, I would like to work on the remaining part of [Dataset and how jobs execute](https://galaxyproject.org/support/how-jobs-execute/) and [A history was shared with me, where can I find it?](https://galaxyproject.org/support/missing-history/). Kindly add my name, if possible!

Done, thanks!

> I also think that we can cover [Checking for active vs deleted vs permanently deleted (purged) datasets and histories](https://galaxyproject.org/learn/managing-datasets/#delete-vs-delete-permanently) under [Managing Datasets](https://galaxyproject.org/learn/managing-datasets/) as it is its sub-part.

Good catch, I've moved it 😺

**Comment by bernandez** [comment]
Hello  @beatrizserrano   i would like to work  on [Getting an account at Galaxy Main (https://usegalaxy.org)](https://galaxyproject.org/support/account/) and 
 [Loading Data](https://galaxyproject.org/support/loading-data/).

**Comment by otatadeabigail** [comment]
Hi @beatrizserrano i would like to work on How to format fastq data for tools that require .fastqsanger format? and "Where is the tool help?"

**Comment by beatrizserrano** [comment]
Updated! Thank you very much @bernandez and @otatadeabigail :)

**Comment by ManaliPawar** [comment]
hello @beatrizserrano , i would like to work on managing datasets (https://galaxyproject.org/learn/managing-datasets/)

**Comment by garimavs** [comment]
Hey @beatrizserrano, the issue ([A history was shared with me, where can I find it?](https://galaxyproject.org/support/missing-history/)) is probably the same as ([My history is missing! Good news, it probably isn't.](https://galaxyproject.org/support/missing-history/#my-history-is-missing-)), which my fellow outreach applicant is probably working on!

**Comment by Melkeb** [comment]
Hey I'm only working on my history is missing and my dataset is missing anything after sharing history I didn't include.

**Comment by garimavs** [comment]
> Hey I'm only working on my history is missing and my dataset is missing anything after sharing history I didn't include.

Oh, thanks for clarifying, will edit my previous text!

**Comment by Beachyesh** [comment]
Hey @beatrizserrano I would like to work on adding the following FAQs to the GTN:
Tool Error? Try sorting your input
Mismatched Chromosome identifiers (and how to avoid them)

**Comment by Beachyesh** [comment]
Hey @beatrizserrano I would like to pick the following FAQs:
[Format help for Tabular/BED/Interval Datasets](https://galaxyproject.org/support/tabular/)
[How do I find, adjust, and/or correct metadata?](https://galaxyproject.org/support/metadata/)

**Comment by beatrizserrano** [comment]
> Hey @beatrizserrano, I would like to merge all **Known Issues** links under one FAQ in GTN to access all known issues easily. I would love to know your opinion!

Merging them sounds good to me. However, I'm wondering whether this FAQ would be really useful to have in the GTN. Any thoughts on this? @jennaj @shiltemann @hexylena

**Comment by shiltemann** [comment]
> > Hey @beatrizserrano, I would like to merge all **Known Issues** links under one FAQ in GTN to access all known issues easily. I would love to know your opinion!
> 
> Merging them sounds good to me. However, I'm wondering whether this FAQ would be really useful to have in the GTN. Any thoughts on this? @jennaj @shiltemann @hexylena

I agree, the average Galaxy/GTN user won't know what to do with the GitHub issues, but that said maybe the list of known issues could be added to the end of the [bug reporting FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting_reporting.html)?

**Comment by garimavs** [comment]
> > > Hey @beatrizserrano, I would like to merge all **Known Issues** links under one FAQ in GTN to access all known issues easily. I would love to know your opinion!
> > 
> > 
> > Merging them sounds good to me. However, I'm wondering whether this FAQ would be really useful to have in the GTN. Any thoughts on this? @jennaj @shiltemann @hexylena
> 
> I agree, the average Galaxy/GTN user won't know what to do with the GitHub issues, but that said maybe the list of known issues could be added to the end of the [bug reporting FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting_reporting.html)?

I see that in [Reporting Usage Issues or Software bugs](https://galaxyproject.org/issues/), the link is provided to these known issues, should I include them while writing about usage problem reporting?

**Comment by otatadeabigail** [comment]
@beatrizserrano please i'd like to work on these:


 [Understanding compressed fastq data (fastq.gz)
 [Reformatting fastq data loaded with NCBI SRA]
 [Common datatypes explained]
 [Tool error? Try Sorting Your Inputs]
 [Preparing and using a Custom Reference Genome or Build]
 [Mismatched Chromosome identifiers (and how to avoid them)]
 [Extended Help for Differential Expression Analysis Tools]

However, kindly help me review the pull request of one of the FAQ's I moved, so i'd be sure I did the right thing before moving on to others. Thanks

**Comment by beatrizserrano** [comment]
> > > > Hey @beatrizserrano, I would like to merge all **Known Issues** links under one FAQ in GTN to access all known issues easily. I would love to know your opinion!
> > > 
> > > 
> > > Merging them sounds good to me. However, I'm wondering whether this FAQ would be really useful to have in the GTN. Any thoughts on this? @jennaj @shiltemann @hexylena
> > 
> > 
> > I agree, the average Galaxy/GTN user won't know what to do with the GitHub issues, but that said maybe the list of known issues could be added to the end of the [bug reporting FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting_reporting.html)?
> 
> I see that in [Reporting Usage Issues or Software bugs](https://galaxyproject.org/issues/), the link is provided to these known issues, should I include them while writing about usage problem reporting?

I am not sure to be honest. I think that this can be hard for users to browse, but @jennaj may have a different view.

**Comment by AfiMaameDufie** [comment]
Hi @beatrizserrano, 
I would like to work on :

[The account usage quota seems incorrect](https://galaxyproject.org/support/account-quotas/) under accounts

and 
 [Known Issues - Galaxy]
 [Known Issues - Main https]
 [Known Issues - Tools-devteam]
 [Known Issues - Tools-iuc]
 
 under Known issues
 
 Can I go ahead?

**Comment by FrozenGirl-spd** [comment]
Hey @beatrizserrano, May I work on:
[Known Issues - Main https://usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook/issues/)
[Known Issues - Tools-devteam](https://github.com/galaxyproject/tools-devteam/issues/)

**Comment by Watiriroyalty** [comment]
Hello @beatrizserrano I would like to work on [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

**Comment by Watiriroyalty** [comment]
@beatrizserrano : I would like to work on
Help Guides
 [Why develop tools for Galaxy?](https://galaxyproject.github.io/why-galaxy)
 [Galaxy Tricks](https://github.com/bgruening/galaxy-tricks/blob/master/README.md)
 [Commercial Galaxy Support](https://galaxyproject.org/support/commercial/)

**Comment by adi-1999** [comment]
@beatrizserrano  please I would like to work on these:
[Known Issues - Main https://usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook/issues/)
 [Known Issues - Tools-devteam](https://github.com/galaxyproject/tools-devteam/issues/)
 [Known Issues - Tools-iuc](https://github.com/galaxyproject/tools-iuc/issues) 
 [Reset password or Change email, username, password](https://galaxyproject.org/support/account-updates/)
 [Why develop tools for Galaxy?](https://galaxyproject.github.io/why-galaxy)
 [Galaxy Tricks](https://github.com/bgruening/galaxy-tricks/blob/master/README.md)
 [Commercial Galaxy Support](https://galaxyproject.org/support/commercial/)
 [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)
 [Moving data between Galaxy servers (any!)](https://genomicsvirtuallab.wordpress.com/2016/07/13/moving-data-between-galaxy-instances/)

**Comment by bernandez** [comment]
Hello @beatrizserrano i would also like to work on
-   [The account usage quota seems incorrect](https://galaxyproject.org/support/account-quotas/)

**Comment by beatrizserrano** [comment]
> Hi @beatrizserrano, I would like to work on :
> 
> [The account usage quota seems incorrect](https://galaxyproject.org/support/account-quotas/) under accounts

Added, thanks! I've divided that one into several, so let me know if you want to work on all the ones under the same page.

> and [Known Issues - Galaxy] [Known Issues - Main https] [Known Issues - Tools-devteam] [Known Issues - Tools-iuc]

We have been discussing leaving those out as they are too technical for users

**Comment by beatrizserrano** [comment]
> Hey @beatrizserrano, May I work on: [Known Issues - Main https://usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook/issues/) [Known Issues - Tools-devteam](https://github.com/galaxyproject/tools-devteam/issues/)

Hi @FrozenGirl-spd! We have decided to skip those FAQs, could you select another one? Thanks!

**Comment by beatrizserrano** [comment]
> Hello @beatrizserrano I would like to work on [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

Hi @Watiriroyalty. This one was marked as done because it's already a tutorial, so no need to create a FAQ. Could you pick another one? Thanks :)

**Comment by beatrizserrano** [comment]
> @beatrizserrano : I would like to work on Help Guides [Why develop tools for Galaxy?](https://galaxyproject.github.io/why-galaxy) [Galaxy Tricks](https://github.com/bgruening/galaxy-tricks/blob/master/README.md) [Commercial Galaxy Support](https://galaxyproject.org/support/commercial/)

these are also marked as done, sorry :(

**Comment by beatrizserrano** [comment]
> @beatrizserrano please I would like to work on these: [Known Issues - Galaxy](https://github.com/galaxyproject/galaxy/issues/) [Known Issues - Main https://usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook/issues/) [Known Issues - Tools-devteam](https://github.com/galaxyproject/tools-devteam/issues/) [Known Issues - Tools-iuc](https://github.com/galaxyproject/tools-iuc/issues) [Reset password or Change email, username, password](https://galaxyproject.org/support/account-updates/) [Why develop tools for Galaxy?](https://galaxyproject.github.io/why-galaxy) [Galaxy Tricks](https://github.com/bgruening/galaxy-tricks/blob/master/README.md) [Commercial Galaxy Support](https://galaxyproject.org/support/commercial/) [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/) [Moving data between Galaxy servers (any!)](https://genomicsvirtuallab.wordpress.com/2016/07/13/moving-data-between-galaxy-instances/)

Hi @adi-1999. All those FAQs are either marked as done or selected by another person. Could you select some free ones? Thanks!

**Comment by garimavs** [comment]
Hey @beatrizserrano, Can I work on [Updating account preferences](https://galaxyproject.org/support/account/#registering-for-a-galaxy-account)

**Comment by beatrizserrano** [comment]
> Hey @beatrizserrano, Can I work on [Updating account preferences](https://galaxyproject.org/support/account/#registering-for-a-galaxy-account)

Sure, assigned! Thanks

**Comment by bernandez** [comment]
Hi @beatrizserrano I had worked on updating account preference. You had assigned this to me. A pull request has been opened for this FAQ.

**Comment by beatrizserrano** [comment]
> Hi @beatrizserrano I had worked on updating account preference. You had assigned this to me. A pull request has been opened for this FAQ.

True! Sorry @bernandez and @garimavs. I'm afraid that there might be more mistakes, it's a bit confusing...

**Comment by adi-1999** [comment]
Hi @beatrizserrano , can I please work on :-
[Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)
[How to log in or out of Galaxy?](https://galaxyproject.org/support/account-quotas/#how-to-log-in-or-out-of-galaxy)
[My job ended with an error. What can I do?](https://galaxyproject.org/support/tool-error/)
[I need more space for my work](https://galaxyproject.org/support/account-quotas/#i-need-more-space-for-my-work)
 [Find all Histories and purge (aka permanently delete)](https://galaxyproject.org/support/account-quotas/#find-all-histories-and-purge-aka-permenently-delete-those-not-needed)
 [Find all Datasets or just review those in the active history and purge (aka permenently delete) those not needed](https://galaxyproject.org/support/account-quotas/#find-all-datasets-or-just-review-those-in-the-active-history-and-purge-aka-permenently-delete-those-not-needed)
 [Find Histories that have been shared with you, and unshare those not needed](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed)
 [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

**Comment by beatrizserrano** [comment]
> Hi @beatrizserrano , can I please work on :- [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/) [How to log in or out of Galaxy?](https://galaxyproject.org/support/account-quotas/#how-to-log-in-or-out-of-galaxy) [My job ended with an error. What can I do?](https://galaxyproject.org/support/tool-error/) [I need more space for my work](https://galaxyproject.org/support/account-quotas/#i-need-more-space-for-my-work) [Find all Histories and purge (aka permanently delete)](https://galaxyproject.org/support/account-quotas/#find-all-histories-and-purge-aka-permenently-delete-those-not-needed) [Find all Datasets or just review those in the active history and purge (aka permenently delete) those not needed](https://galaxyproject.org/support/account-quotas/#find-all-datasets-or-just-review-those-in-the-active-history-and-purge-aka-permenently-delete-those-not-needed) [Find Histories that have been shared with you, and unshare those not needed](https://galaxyproject.org/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed) [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

Updated, thanks! But please note that some of them are already done

**Comment by garimavs** [comment]
> > Hi @beatrizserrano I had worked on updating account preference. You had assigned this to me. A pull request has been opened for this FAQ.
> 
> True! Sorry @bernandez and @garimavs. I'm afraid that there might be more mistakes, it's a bit confusing...

Oh no worries!

**Comment by AnomalyCodes** [comment]
Hi @beatrizserrano. I would like to work on [Transfer/Upload entire histories] Please assign it to me as I can see that it's available. Thank you.

**Comment by AnomalyCodes** [comment]
Hi @beatrizserrano. I would like to work on [Transfer/Upload entire histories](https://galaxyproject.org/support/download-data/#overview) Please assign it to me as I can see that it's available. Thank you.

**Comment by bernandez** [comment]
Hi @beatrizserrano I would also like to work on  [Reduce quota usage while still retaining prior work (data, tools, methods)](Reduce quota usage while still retaining prior work (data, tools, methods)). Thank you.

**Comment by garimavs** [comment]
Hey @otatadeabigail, I would like to work with you on [Understanding compressed fastq data (fastq.gz)](https://galaxyproject.org/support/compressed-fastq/) and [Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/), unless you have already submitted a pr for the mentioned.

**Comment by Samanthaanjei** [comment]
hello @beatrizserrano right now I m working on DEVELOPING A CULTURE OF WELLBEING FOR OPEN-SOURCE PROJECTS – THE GALAXY COMMUNITY AS A USE CASE.
[PROJECT 1.docx](https://github.com/galaxyproject/galaxy-hub/files/8410233/PROJECT.1.docx)

for now I have been to answer the first part of the questions but in order to proceed answering the rest i will need to ask you a few questions concerning the Galaxy Community.
thank you

**Comment by bernandez** [comment]
Hello @Samanthaanjei welcome. I would love to help you out on this.

**Comment by Samanthaanjei** [comment]
Thank you very much @bernandez. Please inorder to proceed I will like to find out about the culture of wellbeing of the galaxy community as a whole that is its systems as well as members

**Comment by annefou** [comment]
> hello @beatrizserrano right now I m working on DEVELOPING A CULTURE OF WELLBEING FOR OPEN-SOURCE PROJECTS – THE GALAXY COMMUNITY AS A USE CASE. [PROJECT 1.docx](https://github.com/galaxyproject/galaxy-hub/files/8410233/PROJECT.1.docx)
> 
> for now I have been to answer the first part of the questions but in order to proceed answering the rest i will need to ask you a few questions concerning the Galaxy Community. thank you

Awseome! for such discussions, this issue may not be the best place. Shall we discuss in our outreachy chat instead?

**Comment by Samanthaanjei** [comment]
Alright @annefou I will 🙏🏿

**Comment by anayo-hub** [comment]
hi @beatrizserrano I would love  to work on https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item

**Comment by desmond-jasper** [comment]
@beatrizserrano [Sharing via Web Link and Publishing an Item](https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item)

**Comment by annefou** [comment]
> hi @beatrizserrano I would love to work on https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item

@anayo-hub I think you can check it and work with @NdzeRegina. @desmond-jasper is also interested.

> @beatrizserrano [Sharing via Web Link and Publishing an Item](https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item)

@desmond-jasper ok. Make sure you discuss together and agree on who is doing what and how. Thanks.

**Comment by anayo-hub** [comment]
@desmond-jasper and  @NdzeRegina, can we talk through the channel

**Comment by desmond-jasper** [comment]
@anayo-hub  @NdzeRegina sure

**Comment by Abraham-Deng** [comment]
Hello colleagues, am seeing the same issues almost being shared by all contributors. but am interested in the following issues;
1. Why develop tools for Galaxy?
 2. Galaxy Tricks
 3. Commercial Galaxy Support
@beatrizserrano Please let me know if there are more different issues not being done by other contributors, so I can choose them instead of those ones. OR you may add me more.

**Comment by beatrizserrano** [comment]
Thanks @Abraham-Deng, we will be happy to discuss this in our chat channel and/or our call next week :)

**Comment by mesode** [comment]
Hello @beatrizserrano I would like to begin working on [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

**Comment by beatrizserrano** [comment]
> Hello @beatrizserrano I would like to begin working on [Understanding the Analysis History](https://galaxyproject.org/tutorials/histories/)

Thanks @mesode. That FAQ is already checked because it's a tutorial, I'll remove it to avoid confusion. Could you pick another one that is not check and coordinate with the person that is taking care?

**Comment by mesode** [comment]
Ok. I'll select another one

**Comment by mesode** [comment]
@beatrizserrano Then I will work on registering for a galaxy account (https://galaxyproject.org/support/account/#registering-for-a-galaxy-account) with @bernandez

**Comment by mesode** [comment]
Thank you ma.

**Comment by mesode** [comment]
I have equally started researching on diversity, well-being and mental
health with respect to open source.
I would love to send a small report on these for you to review tomorrow if
it's okay with you.

**Comment by beatrizserrano** [comment]
> I have equally started researching on diversity, well-being and mental health with respect to open source. I would love to send a small report on these for you to review tomorrow if it's okay with you.

As we indicated in the guidelines, we invite you to write your thoughts in a post that you can publish and share as an Outreachy contribution. We are also hosting a call next week, please check the chat channel for details and further discussion. Let's try to keep this issue only for discussions about the FAQs and move the other discussions to our chat.

**Comment by mesode** [comment]
Ok. Got it.

**Comment by Samanthaanjei** [comment]
Hi @beatrizserrano i will like to work on [importing data via links](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_import_via_link.html)

**Comment by Samanthaanjei** [comment]
Hello @adi-1999 can i please work with you on your FAQ

**Comment by Samanthaanjei** [comment]
> Hello @adi-1999 can i please work with you on your FAQ

please @annefou can i collaborate with @adi-1999 on his FAQ's

**Comment by annefou** [comment]
> > Hello @adi-1999 can i please work with you on your FAQ
> 
> please @annefou can i collaborate with @adi-1999 on his FAQ's

Of course you can collaborate. You do not need our permissions to do so. Collaboration is encourage in open source projects.

**Comment by Nurzhamalyrys** [comment]
Hi @ManaliPawar! 
Can I work on managing dataset  issue with you?

**Comment by andreasSune** [comment]
Hi @annefou  please i would like to work on the following issues with the other contributors:[Dataset downloading incompletely?](https://galaxyproject.org/support/download-data/#large-datasets)
[Managing Datasets](https://galaxyproject.org/learn/managing-datasets/)
[Sharing via Web Link and Publishing an Item](https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item)
[I need more space for my work](https://galaxyproject.org/support/account-quotas/#i-need-more-space-for-my-work)
[Determining the job failure type](https://galaxyproject.org/support/tool-error/#determining-the-job-failure-type)
[Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/)
[Extended Help for Differential Expression Analysis Tools](https://galaxyproject.org/support/diff-expression/)

**Comment by Nurzhamalyrys** [comment]
Hi ! @adi-1999 
Can I work on " I need more space on my work" as well ?

**Comment by bernandez** [comment]
Hi @beatrizserrano looks like a pull request has not been open for [How to log in or out of Galaxy?](https://galaxyproject.org/support/account-quotas/#how-to-log-in-or-out-of-galaxy) and 
 [I need more space for my work](https://galaxyproject.org/support/account-quotas/#i-need-more-space-for-my-work). i would love to work on this with @adi-1999.

**Comment by annefou** [comment]
> Hi @beatrizserrano looks like a pull request has not been open for [How to log in or out of Galaxy?](https://galaxyproject.org/support/account-quotas/#how-to-log-in-or-out-of-galaxy) , [Preparing and using a Custom Reference Genome or Build](https://galaxyproject.org/learn/custom-genomes/) and [Mismatched Chromosome identifiers (and how to avoid them)](https://galaxyproject.org/support/chrom-identifiers/). I would love to work on this with @adi-1999.

We have PR for registration, login, etc to galaxy. And this is also covered by the training materials. I am not sure for the two other ones. These are very specific and may be covered in a training.

**Comment by annefou** [comment]
> Hi @annefou please i would like to work on the following issues with the other contributors:[Dataset downloading incompletely?](https://galaxyproject.org/support/download-data/#large-datasets) [Managing Datasets](https://galaxyproject.org/learn/managing-datasets/) [Sharing via Web Link and Publishing an Item](https://galaxyproject.org/learn/share/#sharing-via-web-link-and-publishing-an-item) [I need more space for my work](https://galaxyproject.org/support/account-quotas/#i-need-more-space-for-my-work) [Determining the job failure type](https://galaxyproject.org/support/tool-error/#determining-the-job-failure-type) [Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/) [Extended Help for Differential Expression Analysis Tools](https://galaxyproject.org/support/diff-expression/)

No need to take several FAQs to move. Take one and work on it. However, before starting anything, check the open PRs, the existing/moved FAQs (https://training.galaxyproject.org/faqs/)

**Comment by bernandez** [comment]
Hi @beatrizserrano I just checked an looks like a PR has not been opened for [Preparing and using a Custom Reference Genome or Build](https://galaxyproject.org/learn/custom-genomes/). Can i work on this?

**Comment by mruqqie** [comment]
Hi @bernandez . Can I work on [Updating account preferences]((https://galaxyproject.org/support/account/#registering-for-a-galaxy-account)) and [Registering for a Galaxy account]((https://galaxyproject.org/support/account/#registering-for-a-galaxy-account)) with you?

**Comment by desmond-jasper** [comment]
@annefou please can you check on this: https://github.com/galaxyproject/training-material/pull/3357

**Comment by garimavs** [comment]
> Hey @otatadeabigail, I would like to work with you on [Understanding compressed fastq data (fastq.gz)](https://galaxyproject.org/support/compressed-fastq/) and [Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/), unless you have already submitted a pr for the mentioned.

Hey @beatrizserrano @annefou, I'm working on [Tool error? Try Sorting Your Inputs](https://galaxyproject.org/support/sort-your-inputs/), kindly add my name to the list too.

**Comment by andreasSune** [comment]
Hi @annefou  and @beatrizserrano  please can you check on this: [https://github.com/galaxyproject/training-material/pull/3374](https://github.com/galaxyproject/training-material/pull/3374)

**Comment by Watiriroyalty** [comment]
Hey ? I would love to collaborate with anyone working on a open issue, I notice that most of the issues have already been allocated to fellow contributors.

**Comment by Beachyesh** [comment]
@beatrizserrano FAQ "Where is the tool help?" has a [PR](https://github.com/galaxyproject/training-material/pull/3322) that seems to have been abandoned. May I work on it?

**Comment by mruqqie** [comment]
Hi @annefou and @beatrizserrano can I work on [Determining the job failure type](https://galaxyproject.org/support/tool-error/#determining-the-job-failure-type)?

**Comment by garimavs** [comment]
> Hi @annefou and @beatrizserrano can I work on [Determining the job failure type](https://galaxyproject.org/support/tool-error/#determining-the-job-failure-type)?

Hello @mruqqie, I'm already working on this issue. I will submit a PR soon as the content is being revised, which took some time. You can most definitely share your inputs in that.

**Comment by mruqqie** [comment]
@garimavs Hi. Can I work with you on it then?

**Comment by mruqqie** [comment]
@annefou @beatrizserrano I'd like to work with @Melkeb on [The tool I'm using does not recognize any input datasets. Why?](https://galaxyproject.org/support/datatypes-and-tools/), if that's okay?

**Comment by Abraham-Deng** [comment]
Dear @adi-1999 , May we collaborate on handling the below issues
Known Issues - Galaxy
Known Issues - Main ht
Known Issues - Tools
Known Issues - Tools-iuc
Why develop tools for Galaxy?
Galaxy Tricks
Commercial Galaxy Support
Understanding the Analysis History
Moving data between Galaxy servers (any!)

**Comment by mruqqie** [comment]
Hi @otatadeabigail can I work with you on [Mismatched Chromosome identifiers (and how to avoid them)](https://galaxyproject.org/support/chrom-identifiers/) and [Common datatypes explained](https://galaxyproject.org/learn/datatypes/)?

**Comment by garimavs** [comment]
Hey @beatrizserrano, Kindly add my name in [Managing Datasets](https://galaxyproject.org/learn/managing-datasets/) FAQ.

**Comment by Beachyesh** [comment]
@garimavs I took it up a while back, let's collaborate! :)

**Comment by garimavs** [comment]
> @garimavs I took it up a while back, let's collaborate! :)

Sure! Let's talk more about it on the community channel.
